analyzers/src/SonarAnalyzer.RuleDescriptorGenerator/Program.cs

```diff
@@ -31,6 +31,7 @@
 using System.Xml.Serialization;
 using Microsoft.CodeAnalysis.Diagnostics;
 using SonarAnalyzer.Common;
+using SonarAnalyzer.Rules;
```

Comment:
There are 3 usings that are only needed for NETCOREAPP. Please move them inside the conditional part.

Reply:
I'll delete most of this file in the next PR. I won't fix it here